Configuration/Geometry/python/GeometryDD4hepExtended2021Reco_cff.py

```diff
@@ -11,6 +11,7 @@
 
 #Tracker
 from RecoTracker.GeometryESProducer.TrackerRecoGeometryESProducer_cfi import *
+from Geometry.TrackerGeometryBuilder.trackerParameters_cff import *
```

Comment:
not an issue created by this PR - however having parallel cffs (with / without DD4hep) _and_ a dd4hep era is a rather error prone situation.. What ensures that these do not get mixed up?

Reply:
agree. The dd4hep-specific configurations should go away as soon as all the sub detectors have era-enabled fragments.